plugin/src/sidebar.ts

```diff
@@ -643,37 +633,38 @@ function createLayer(dataset: DataCatalogItem, overrides?: any) {
               },
             },
           }
-        : null,
-      ...(overrides !== undefined
-        ? merge(defaultOverrides, omit(overrides, "data"))
-        : format === ("geojson" || "czml")
-        ? defaultOverrides
-        : format === "gtfs"
-        ? proxyGTFS(overrides)
-        : format === "mvt"
-        ? {
-            polygon: {},
-          }
-        : format === "wms"
-        ? {
-            raster: {
-              alpha: 0.8,
-            },
-          }
-        : { ...(overrides ?? {}) }),
-    },
-    {
-      infobox: {
-        property: {
-          default: {
-            unselectOnClose: true,
+        : {},
+      overrides?.infobox ?? {},
+      infoboxGlobal,
+    ),
+    ...(overrides !== undefined
+      ? merge(defaultOverrides, omit(overrides, ["data", "infobox"]))
+      : format === ("geojson" || "czml")
+      ? defaultOverrides
+      : format === "gtfs"
+      ? proxyGTFS(overrides)
+      : format === "mvt"
+      ? {
+          polygon: {},
+        }
+      : format === "wms"
+      ? {
+          raster: {
+            alpha: 0.8,
           },
-        },
-      },
-    },
-  );
+        }
+      : { ...(omit(overrides, "infobox") ?? {}) }),
```

Comment:
this should be omit(overrides, ["data", "infobox"]) too right?

Reply:
Updated.